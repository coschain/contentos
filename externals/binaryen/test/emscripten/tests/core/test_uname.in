#include <stdio.h>
#include <sys/utsname.h>

int main() {
  struct utsname u;
  printf("ret: %d\n", uname(&u));
  printf("sysname: %s\n", u.sysname);
  printf("nodename: %s\n", u.nodename);
  printf("release: %s\n", u.release);
  printf("version: %s\n", u.version);
  printf("machine: %s\n", u.machine);
  printf("invalid: %d\n", uname(0));
  return 0;
}
