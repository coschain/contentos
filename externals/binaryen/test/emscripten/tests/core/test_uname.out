ret: 0
sysname: Emscripten
nodename: emscripten
release: 1.0
version: #1
machine: x86-JS
